### PR TITLE
Update Code_Guidelines.rst

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: doc/source/Resources/Code_Guidelines.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ Imports
 ~~~~~~~
 Imports should always be placed at the top of the file, just after any
 module comments and docstrings and before module globals and
-constants.  This reduces the likelihood of an ``ImportError`` that may only
+constants.  This reduces the likelihood of an import error that might only
```
Comment (akaszynski) [inline doc/source/Resources/Code_Guidelines.rst:30]: I'd prefer keeping it as ``ImportError``.

File: doc/source/Resources/Code_Guidelines.rst [hunk 2]
```diff
@@ -435,15 +434,18 @@ a module, function, class, or method definition.  A docstring becomes
...
 Write docstrings for all public modules, functions, classes, and
-methods. Docstrings are not necessary for non-public methods, but you
-should have a comment that describes what the method does
+methods. Docstrings are not necessary for non-public methods, but such
+methods should have comments that describe what they do.
 
 To create a docstring, surround the comments with three double quotes
 on either side.
 
 For a one-line docstring, keep both the starting and ending """ on the
```
Comment (akaszynski) [inline doc/source/Resources/Code_Guidelines.rst:443]:
```suggestion
For a one-line docstring, keep both the starting and ending ``"""`` on the
```

File: doc/source/Resources/Code_Guidelines.rst [hunk 3]
```diff
@@ -435,15 +434,18 @@ a module, function, class, or method definition.  A docstring becomes
...
-same line. For example, """This is a docstring.""".  For a multi-line
-docstring, put the ending """ on a line by itself.
+same line. For example:
+
+"""This is a docstring.""".
```
Comment (akaszynski) [inline doc/source/Resources/Code_Guidelines.rst:446]:
```suggestion

.. code:: python

   """This is a docstring.""".  
```

File: doc/source/Resources/Code_Guidelines.rst [hunk 4]
```diff
@@ -435,15 +434,18 @@ a module, function, class, or method definition.  A docstring becomes
...
 on either side.
 
 For a one-line docstring, keep both the starting and ending """ on the
-same line. For example, """This is a docstring.""".  For a multi-line
-docstring, put the ending """ on a line by itself.
+same line. For example:
+
+"""This is a docstring.""".  
+
+For a multi-line docstring, put the ending """ on a line by itself.
```
Comment (akaszynski) [inline doc/source/Resources/Code_Guidelines.rst:448]:
```suggestion
For a multi-line docstring, put the ending ``"""`` on a line by itself.
```

File: doc/source/Resources/Code_Guidelines.rst [hunk 5]
```diff
@@ -648,7 +652,7 @@ within one single code block.  For example:
 
 
 Aside from the lack of comments, this complex nested validation method
-is difficult to debug (and validate with unit testing).  It would
+is difficult to debug and validate with unit testing. It would
 be far better to implement more validation methods, join conditionals,
 and so on.
```
Comment (akaszynski) [inline doc/source/Resources/Code_Guidelines.rst:657]:
```suggestion
etc.
```

File: doc/source/Resources/Code_Guidelines.rst [hunk 6]
```diff
@@ -718,7 +722,7 @@ documented.  Additionally, while it's a trivial example, you could
 implement a unit test for ``is_consonant``.
 
 
-AEDT-Specific Coding Guidelines
+PyAEDT-Specific Coding Guidelines
 -------------------------------
```
Comment (akaszynski) [inline doc/source/Resources/Code_Guidelines.rst:726]:
```suggestion
---------------------------------
```

File: doc/source/Resources/Code_Guidelines.rst [hunk 7]
```diff
@@ -727,18 +731,18 @@ Logging Errors
...
 .. code:: python
 
     self.messenger.add_error_message("This is an error message.")
     self.messenger.add_warning_message("This is a warning message.")
     self.messenger.add_info_message("This is an info message.")
 
-The above messages are written to both AEDT message windows and the
-log file. If you want the message to be written only to the log file,
-use:
+These exmaples demonstratw how to to write messages only to the log file:
```
Comment (akaszynski) [inline doc/source/Resources/Code_Guidelines.rst:745]:
```suggestion
These examples demonstrate how to to write messages only to the log file:
```

File: doc/source/Resources/Code_Guidelines.rst [hunk 8]
```diff
@@ -792,5 +797,5 @@ log file. Here is an example of an error:
 
 Hard-Coding Values
 ~~~~~~~~~~~~~~~~~~
-Do not write to the registry hard-coded values that the code is to
-use. Instead, use the Configuration service.
+Do not write hard-coded values that the code is to
+use to the registry. Instead, use the Configuration service.
```
Comment (akaszynski) [inline doc/source/Resources/Code_Guidelines.rst:801]:
```suggestion
Do not write hard-coded values to the registry. Instead, use the Configuration service.
```

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.

File: doc/source/Resources/Code_Guidelines.rst (hunk 1 above)
Comment (PipKat) [inline doc/source/Resources/Code_Guidelines.rst:30]:
```suggestion
constants.  This reduces the likelihood of an ``ImportError`` that might only
```

File: doc/source/Resources/Code_Guidelines.rst [hunk 9]
```diff
@@ -648,7 +652,7 @@ within one single code block.  For example:
 
 
 Aside from the lack of comments, this complex nested validation method
-is difficult to debug (and validate with unit testing).  It would
+is difficult to debug and validate with unit testing. It would
 be far better to implement more validation methods, join conditionals,
```
Comment (PipKat) [inline doc/source/Resources/Code_Guidelines.rst:656]:
```suggestion
be far better to implement more validation methods and join conditionals.
```

File: doc/source/Resources/Code_Guidelines.rst (hunk 5 above)
Comment (PipKat) [inline doc/source/Resources/Code_Guidelines.rst:657]:
```suggestion
```

**Pull request merged**